concrete/src/Application/UserInterface/Menu/Item/Item.php

```diff
@@ -17,12 +17,12 @@ public function __construct($handle, $pkgHandle = false)
 
     public function getHandle()
     {
-        return $this->handle;
+        return isset($this->handle) ? $this->handle : null;
```

Comment:
I know that, but would you create them as `protected` (but it'd be a non BC change) or `public` (that's an invitation to use the properties instead of the getters)?